Add backlog for new submitters to monitor (#368)
hacs/documentation: documentation/publish/include.md

### Lint [sorted]

This make sure that the files still are sorted correctly.

## After Submitting

> HACS contains a significant amount of repos with new repos added on almost a daily basis. Check the [backlog](https://github.com/hacs/default/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc+draft%3Afalse) before querying your PR. Note that leaving a comment on your PR will drop it to the bottom of the list.   

Reviewer: ```suggestion
HACS contains a significant amount of repos, with new repositories added almost daily. Check the [backlog](https://github.com/hacs/default/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc+draft%3Afalse) before querying your PR. Updating your PR will drop it to the bottom of the list.   

```
